test/fixtures/combined_vancouver_queens.json

```diff
@@ -892460,30 +892292,6 @@
       "source_id": "node/3404829999"
     }
   },
-  {
-    "_index": "pelias",
-    "_id": "openstreetmap:venue:node/3428101636",
-    "_type": "_doc",
-    "data": {
-      "name": {
-        "default": "Vancouver Pacific Central Station"
```

Comment:
This one would have been nice since the name is fairly unique.
Fortunately that node has since been deleted and replaced with https://www.openstreetmap.org/way/32603955 which we will bring in via `building~name`